functions/src/http.ts

```diff
@@ -0,0 +1,22 @@
+import * as request from 'request';
```

Comment:
since you're importing a new lib, you need to update /functions/package.json as well (which is what firebase will use when the app is deployed)